diff --git a/pytest.ini b/pytest.ini
index 825dbe6..40efb05 100644
--- a/pytest.ini
+++ b/pytest.ini
@@ -20,3 +20,6 @@ filterwarnings =
     ignore:Unicode unequal comparison failed to convert:UnicodeWarning
     # https://github.com/pypa/setuptools/issues/2025
     ignore:direct construction of .*Item has been deprecated:DeprecationWarning
+    # https://github.com/pypa/setuptools/issues/2081
+    ignore:lib2to3 package is deprecated:PendingDeprecationWarning
+    ignore:lib2to3 package is deprecated:DeprecationWarning
