diff -U3 -r setuptools-49.6.0-orig/pytest.ini setuptools-49.6.0/pytest.ini
--- setuptools-49.6.0-orig/pytest.ini	2020-08-20 14:51:59.289846443 +0200
+++ setuptools-49.6.0/pytest.ini	2020-08-20 14:52:34.089859558 +0200
@@ -1,5 +1,5 @@
 [pytest]
-addopts=--doctest-modules --flake8 --doctest-glob=pkg_resources/api_tests.txt --cov -r sxX
+addopts=--doctest-modules --doctest-glob=pkg_resources/api_tests.txt -r sxX
 norecursedirs=dist build *.egg setuptools/extern pkg_resources/extern pkg_resources/tests/data tools .* setuptools/_vendor pkg_resources/_vendor
 doctest_optionflags=ELLIPSIS ALLOW_UNICODE
 filterwarnings =
